client/packages/invoices/src/Returns/InboundDetailView/Footer/Footer.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { FC, memo } from 'react';
+import {
+  Box,
+  ButtonWithIcon,
+  StatusCrumbs,
+  XCircleIcon,
+  useTranslation,
+  AppFooterPortal,
+  useBreadcrumbs,
+  InvoiceNodeStatus,
+} from '@openmsupply-client/common';
+import {
+  getStatusTranslator,
+  inboundReturnStatuses,
+  outboundStatuses,
+} from '../../../utils';
+import { InboundReturnFragment, useReturns } from '../../api';
+import { StatusChangeButton } from './StatusChangeButton';
+import { OnHoldButton } from './OnHoldButton';
+
+const createStatusLog = (invoice: InboundReturnFragment) => {
+  const statusIdx = outboundStatuses.findIndex(s => invoice.status === s);
+  const statusLog: Record<InvoiceNodeStatus, null | undefined | string> = {
+    [InvoiceNodeStatus.New]: null,
+    [InvoiceNodeStatus.Picked]: null,
+    [InvoiceNodeStatus.Shipped]: null,
+    [InvoiceNodeStatus.Delivered]: null,
+    [InvoiceNodeStatus.Verified]: null,
```

Comment:
I don't think we want "Picked" or "Shipped" for Inbounds, do we?

Reply:
Maybe shipped, but probably not Picked?

Reply:
Went with this as that's what we have for Inbound Shipments (I think we pass through the Picked status from connected Outbound so we can see progress??)

Though after the conversation about maybe there being no `Picked` status on the Outbound Return... might wait for confirmation from Richard on what these will be?